app-frontend/src/app/pages/projects/edit/browse/browse.html

```diff
@@ -43,33 +43,59 @@ <h5 class="sidebar-title">Browse Scenes</h5>
     </li>
   </ul>
 </div>
-<div class="list-group" ng-if="$ctrl.lastSceneResult && $ctrl.projectScenesReady">
+<div class="list-group" ng-if="$ctrl.isRfScene && $ctrl.lastSceneResult && $ctrl.projectScenesReady">
   <div class="list-group-item">
     <strong class="color-dark">{{$ctrl.lastSceneResult.count | number}} scenes found</strong>
   </div>
 </div>
+<div class="list-group" ng-if="!$ctrl.isRfScene && $ctrl.isLoadingPlanetScenes">
+  <div class="list-group-item">
+    <strong class="color-dark">Loading Planet Scenes...</strong>
+  </div>
+</div>
+<div class="list-group" ng-if="!$ctrl.isRfScene && $ctrl.planetSceneCounts && !$ctrl.isLoadingPlanetScenes">
+  <div class="list-group-item" ng-if="$ctrl.hasMorePlanetPages">
+    <strong class="color-dark">{{$ctrl.planetSceneCounts | number}}+ scenes found</strong>
+  </div>
+  <div class="list-group-item" ng-if="!$ctrl.hasMorePlanetPages">
+    <strong class="color-dark">{{$ctrl.planetSceneCounts | number}} scenes found</strong>
+  </div>
+</div>
 <div class="sidebar-scrollable list-group">
   <rf-scene-item
-      ng-click="$ctrl.openSceneDetailModal(scene)"
       scene="scene"
       selected="$ctrl.isSelected(scene)"
       is-disabled="$ctrl.isInProject(scene)"
+      is-rf-scene="$ctrl.isRfScene"
+      planet-key="$ctrl.planetKey"
       on-select="$ctrl.setSelected(scene, selected)"
+      ng-click="$ctrl.openSceneDetailModal(scene)"
       ng-mouseenter="$ctrl.setHoveredScene(scene)"
       ng-mouseleave="$ctrl.removeHoveredScene()"
       ng-repeat="scene in $ctrl.sceneList track by scene.id">
   </rf-scene-item>
+  <img ng-attr-src="{{$ctrl.base64Uri}}">
   <div class="sidebar-content">
     <button class="btn btn-block btn-secondary"
-            ng-if="!$ctrl.loadingScenes && $ctrl.lastSceneResult && $ctrl.lastSceneResult.hasNext && !$ctrl.errorMsg"
+            ng-if="$ctrl.sourceRepo === 'Raster Foundry' && !$ctrl.loadingScenes && $ctrl.lastSceneResult && $ctrl.lastSceneResult.hasNext && !$ctrl.errorMsg"
```

Comment:
This is large enough that I'd do it as a function in the controller, called in the ng-if to return a boolean.
My personal rule of thumb is if you have more than 3 chained boolean values, it can probably be simplified or refactored to be more clear, even if it's just sticking it in a named function to make it clear what it's evaluating.